src/api/app/assets/javascripts/webui/request.js

```diff
@@ -135,9 +135,6 @@ $(document).ready(function(){
     }
   });
   // TODO: Remove the enclosing code when the request_show_redesign feature is finished - END
-  $('#changes-item').on('shown.bs.tab', function () {
```

Comment:
The `request_show_redesign` feature is finished already? That's why we remove this code?

Reply:
No, but we don't need this code anymore after we introduced our changes. The reason is that we no longer call Ajax but we load the page and display the content of one action at a time.